content/english/post/confederacy-decatur.md

```diff
@@ -11,6 +11,10 @@ tags = ["municipal-action"]
 weight = 1
 +++
 
-A 30-foot obelisk Confederate moment, which has stood for 112 years, was taken down in the downtown Decatur square, after a DeKalb county judge ordered it taken down.
+A 30-foot obelisk Confederate moment, which has stood for 112 years, was taken down in the downtown Decatur square, after a DeKalb county judge ordered it taken down.[^1]
 
 Read more here: https://www.ajc.com/news/local/breaking-confederate-monument-decatur-appears-coming-down/1SfeR7g7YZdScfGI5NVfSJ/
```

Comment:
This should be removed since it's same link as footnote

Reply:
I think "Read more here:" is in a few other posts. Should we change those as well?